README.md

```diff
@@ -6,97 +6,128 @@ The goal of this project is to empower citizens to effectively organize for thei
 
-Download code and dependencies
+Connect to your MySQL instance:
 
 ```bash
-npm install knex -g
-git clone git@github.com:meirim-org/meirim.git
-cd meirim/
-npm i
+$ mysql -uroot -p
 ```
 
-Install chrome dependencies (for ubuntu):
+Create a database for the project:
 
-```bash
- apt-get install -yq gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget
- ```
+```sql
+CREATE DATABASE meirim character set UTF8 collate utf8_bin;
+exit;
+```
 
-Import the database schema
+Edit the local configuration file and set your database and email settings (if needed):
 
 ```bash
-mysql -uroot -p
+$ vi config/local.json
 ```
 
-Create a new schema:
+Install knex globally and run all migrations:
+```bash
+$ npm install knex -g
+$ knex migrate:latest
+```
 
-```sql
-CREATE DATABASE meirim character set UTF8 collate utf8_bin;
-exit;
+Finally, run the service:
+
+```bash
+$ npm run api
 ```
 
-Run migrations
+The service will then be available on port 3001.
+
+### Instructions for frontend
+
+Nothing needs to be set up specifically for the frontend (however a working backend service would make it a bit more useful). It can be started using:
 
 ```bash
-knex  migrate:latest
+$ npm start
 ```
 
-Edit the local configuration file and set your database and email settings
+And will then be available at [http://localhost:3000](http://localhost:3000)
+
+### Running both backend and frontend
+
+Both the api and the frontend need to be run separately when developing for the auto-reload capabilities of webpack-dev-server.
+The api will run on port 3001 by default, and the frontend will run on port 3000 and proxy requests destined to the api from any path beginning with "/api" to the service at port 3001 (proxy settings live in [src/setupProxy.js](src/setupProxy.js)).
 
 ```bash
-vi config/local.json
+$ npm run api
+$ npm start
 ```
 
-## Running in production
+### Instructions for crawler
 
-Run Server in production
+To run the crawler (for testing or just seeding the database with plan data) you must first install Chromium dependencies for your system (these are for Ubuntu):
 
 ```bash
-npm run build
-pm2 start ecosystem.config.js --env production
+$ apt-get install -yq gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils
```

Comment:
This command will not work on Windows as well.

Reply:
Is this stuff even needed?

Reply:
@TomerMe2 @dortheimer  
It is not needed for development, only for deployment

Reply:
they're probably still needed for running Chromium (by puppeteer).
since we all have our desktops set up and are probably already using some browser or another we probably have these (or equivalents) installed already, but if we set up a new server or container these will probably still be needed.

@TomerMe2 this is only an example and only for Ubuntu, so this wouldn't work on many machines. I don't know if we should spend the effort of adding instructions for setting this up on any os and distro, but maybe add a link to the puppeteer website?

Reply:
i removed this sample command and added a link to the puppeteer repo